packages/cozy-sharing/src/components/ShareDialogCozyToCozy.jsx

```diff
@@ -40,7 +40,9 @@ const ShareDialogCozyToCozy = ({
       disableEnforceFocus
       open={true}
       onClose={onClose}
-      title={t(`${documentType}.share.title`, { name: document.name })}
+      title={t(`${documentType}.share.title`, {
+        name: document.name || (document.attributes && document.attributes.name)
```

Comment:
`?.` is available on this project so I think you can use `document.attributes?.name`.

Reply:
I didn't know, it's strange it's not available in all projects (can't remember which one but I'm 100% sure it was breaking)